.github/workflows/tests.yml

```diff
@@ -195,11 +195,11 @@ jobs:
       matrix:
         include:
           - runner: "13-arm64-${{github.run_id}}-${{github.run_attempt}}"
+          - runner: "13-${{github.run_id}}-${{github.run_attempt}}"
           - runner: "12-arm64"
           - runner: "12-${{github.run_id}}-${{github.run_attempt}}"
           - runner: "11-arm64"
           - runner: "11-${{github.run_id}}-${{github.run_attempt}}"
-          - runner: "10.15-${{github.run_id}}-${{github.run_attempt}}"
```

Comment:
Does this require any announcement?

Reply:
@MikeMcQuaid wanted to make an announcement when we had both Intel and ARM Ventura fully supported, so I suppose that would be now. I'm not sure in the past we've made an explicit announcement about support being dropped from unsupported macOS versions.

Reply:
The Catalina runner completes it's depreciation process on 12/1, so may be worth mentioning in that same post.

Reply:
Good pint - looks like the extended stay of 10.15 has been a good thing. Breaking GitHub Actions' `macos-10.15` would probably have been an absolute mess.

Reply:
We'll announced this in the next major or minor version 👍🏻